packages/explorer/src/helpers/postMessageRelayHelpers.ts

```diff
@@ -114,7 +115,7 @@ export type OutgoingEmbedMessage =
       name: typeof EXPLORER_SUBSCRIPTION_RESPONSE;
       operationId: string;
       response: {
-        data?: ExecutionResult<JSONObject>;
+        data?: ExecutionResult<ObjMap<unknown>> | ExecutionResult<JSONValue>;
```

Comment:
Does it work if we just do | on the values basted to execution result? 
```suggestion
        data?: ExecutionResult<JSONValue | ObjMap<unknown>>;
```

Reply:
yes!